help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,119 @@
+
+![Authentication screenshot](../../assets/catalog/analytics/gainsight_px/auth-screen.png)
+* **[!UICONTROL Password]**: The password used to login to [!DNL Gainsight PX](https://app.aptrinsic.com)
+* **[!UICONTROL Client ID]**: The Gainsight PX subscription ID on the [Company Details page](https://app.aptrinsic.com/settings/subscription)
+* **[!UICONTROL Client secret]**: The OAuth secret generated at the bottom of the [Company Details page](https://app.aptrinsic.com/settings/subscription) in the [!DNL Gainsight PX] UI.
+* **[!UICONTROL Username]**: The email used to log in to the [!DNL Gainsight PX](https://app.aptrinsic.com) UI
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination details screen in the Experience Platform user interface showing how to fill in the Name and Description fields](../../assets/catalog/analytics/gainsight_px/destination_details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>* To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map identities {#map}
+This destination supports the mapping of profile attributes and identity namespaces. The target mapping must always be the IDENTIFY_ID identity namespace.
+
+See the examples below to better understand how to configure mapping.
+#### Map a profile attribute {#map-profile-attribute)
+In the example shown below, the source field is an XDM profile attribute which gets mapped to the IDENTIFY_ID target namespace.
```

Comment:
The screenshots and the mapping sections do not match. Please review the content and switch the screenshots to match the sections.

Reply:
good catch, thanks
fixed